plugins/katello/3.14/installation/index.md

```diff
@@ -170,6 +170,74 @@ arguments. Multiple entries can be entered via
         reverse: true
         contact: hostmaster.example.com.
 
+## Tuning options
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from Foreman environments deployed at scale in large user environments.
+
+When the foreman-installer is run, it is deployed with a predefined tuning profile named `default`. The definition of the `default` profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
+
+Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Based on your environment needs, use one of the tuning profiles in the installer as shown below.
+
+```bash
+foreman-installer --tuning medium
+```
+
+```bash
+foreman-installer --tuning large
+```
+
+```bash
+foreman-installer --tuning extra-large
+```
+
+```bash
+foreman-installer --tuning extra-extra-large
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+**Note**
+
+- Using the `--tuning` option does not update `/etc/foreman-installer/custom-hiera.yml`, instead it directly updates the required configuration as specified in the corresponding tuning profile. You can still use `custom-hiera.yml` to override any configuration if really needed.
+- If you had already used `custom-hiera.yml` and starting to use the tuned profiles, you may want to review the definition of tuned profiles (`/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`) and remove the duplicated configuration entries from your `custom-hiera.yml`.
+- Use `foreman-installer --help | grep tuning` to identify the current tuning level.
+
+Sample output for `default` tuning:
+
+```bash
```

Comment:
On Github you can use `console` as well and use `#` (or `$`, but this should run as root) to separate the command from output:
```console
# foreman-installer --help | grep tuning
     --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "default")
```

Don't know if our website renders it in the same way.

Reply:
yes I have seen it in other projects.  But I don't see any reference of it being used in this project. So I used the `bash` format.